samples/client/petstore/java/resttemplate-withXml/src/main/java/org/openapitools/client/model/AdditionalPropertiesClass.java

```diff
@@ -144,6 +144,9 @@ public AdditionalPropertiesClass putMapStringItem(String key, String mapStringIt
   @javax.annotation.Nullable
   @JsonProperty(JSON_PROPERTY_MAP_STRING)
   @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
+  // xmlName=
```

Comment:
looks like xmlName is complete empty. Is that expected?

Reply:
I added a commit to make this change.